apps/nextjs-website/src/components/atoms/UserInfo/UserInfo.tsx

```diff
@@ -1,24 +1,53 @@
 'use client';
-import { translations } from '@/_contents/translations';
-import { PersonOutline } from '@mui/icons-material';
-import { Button, Stack, Hidden, Typography } from '@mui/material';
+import { Login, Logout } from '@mui/icons-material';
+import ArrowDropDownOutlinedIcon from '@mui/icons-material/ArrowDropDownOutlined';
+import {
+  Stack,
+  Typography,
+  IconButton,
+  Avatar,
+  Menu,
+  MenuItem,
+  ListItemIcon,
+  useTheme,
+  Link as MuiLink,
+} from '@mui/material';
 import { Auth } from 'aws-amplify';
-import { FC, useCallback } from 'react';
+import { FC, useCallback, useState } from 'react';
 import { useUser } from '@/helpers/user.helper';
+import Link from 'next/link';
+import { useTranslations } from 'next-intl';
+import { useRouter, usePathname } from 'next/navigation';
+import { isProduction } from '@/config';
 
 const UserInfo: FC = () => {
-  const {
-    auth: { logout },
-  } = translations;
+  const t = useTranslations();
+  const router = useRouter();
+  const pathname = usePathname();
   const { user } = useUser();
```

Comment:
relative to comment below
```suggestion
  const { user, loading } = useUser();
```